diagnostic.md

```diff
@@ -32,8 +40,10 @@ y = 10;
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- solution below -->
+x = y
+y = 10
```

Comment:
yes, `x = y`, but what did y equal at the time? `x = 3` here (it picks up y's value)